src/main/java/wooteco/subway/line/service/LineService.java

```diff
@@ -0,0 +1,100 @@
+package wooteco.subway.line.service;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import wooteco.subway.line.dao.LineDao;
+import wooteco.subway.line.domain.Line;
+import wooteco.subway.line.dto.LineCreateRequest;
+import wooteco.subway.line.dto.LineResponse;
+import wooteco.subway.line.dto.LineUpdateRequest;
+import wooteco.subway.station.dao.StationDao;
+import wooteco.subway.station.domain.Station;
+
+@Service
+public class LineService {
+    private final LineDao lineDao;
+    private final StationDao stationDao;
+
+    public LineService(LineDao lineDao, StationDao stationDao) {
+        this.lineDao = lineDao;
+        this.stationDao = stationDao;
+    }
+
+    @Transactional
+    public LineResponse save(LineCreateRequest lineCreateRequest) {
+        validateDuplicateName(lineCreateRequest.getName());
+        validateIfDownStationIsEqualToUpStation(lineCreateRequest);
+
+        Line line = Line.of(lineCreateRequest);
```

Comment:
```suggestion
        Line line = lineCreateRequest.toLine();
```
Line에서 DTO의 존재를 알 필요가 없어요.
오히려 Request DTO에서 Line 생성에 대한 정보를 다 알고 있으니까요 :)